src/tests/__snapshots__/Header.test.js.snap

```diff
@@ -0,0 +1,23 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Header shows theme buttons 1`] = `
+<header
+  class="App-header"
+>
+  <h1>
+    Blog Demo
+  </h1>
+  <div>
+    <button
+      type="button"
+    >
+      light
+    </button>
+    <button
+      type="button"
+    >
+      dark
+    </button>
```

Comment:
Minor readability suggestion
```suggestion
    <button type="button">
      light
    </button>
    <button type="button">
      dark
    </button>
```

Reply:
this is a snapshot, generated file, so there is no point on amending it.

Reply:
This is a snapshot, generated file, so there is no point on amending it.

Reply:
👍